content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -14,31 +14,54 @@ Role-based access control (RBAC) is a method of regulating access to computer or
 {{% /capture %}}
 
 {{% capture body %}}
-`RBAC` uses the `rbac.authorization.k8s.io` {{< glossary_tooltip text="API Group" term_id="api-group" >}}
-to drive authorization decisions, allowing admins to dynamically configure policies
-through the Kubernetes API.
+RBAC authorization uses the `rbac.authorization.k8s.io`
+{{< glossary_tooltip text="API group" term_id="api-group" >}} to drive authorization
+decisions, allowing you to dynamically configure policies through the Kubernetes API.
 
 As of 1.8, RBAC mode is stable and backed by the rbac.authorization.k8s.io/v1 API.
 
-To enable RBAC, start the apiserver with `--authorization-mode=RBAC`.
+To enable RBAC, start the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+with `--authorization-mode=RBAC`.
```

Comment:
Because line 23 is instructing the reader to run the kube-apiserver with args, you should refer to the actual name of the server (not API server).

To enable RBAC in Kubernetes, start the API server as `kube-apiserver --authorization-mode=RBAC` OR
To enable RBAC mode in Kubernetes, run the API server as follows:
```
kube-apiserver --authorization-mode=RBAC
```

Reply:
I'll also mention other authorization modes.